content/use-cases/defi/algorithmic-trading.md

```diff
@@ -0,0 +1,89 @@
+---
+html: algorithmic-trading.html
+parent: defi-uc.html
+blurb: The XRP Ledger's decentralized exchange consists of an unlimited number of currency pairs, tracked on-demand when users make trades. 
+labels:
+  - Transactions
+---
+# Algorithmic Trading
+
+The XRP Ledger's decentralized exchange presents an opportunity to earn money through _algorithmic trading_, which means running a computer program to find and take profitable trading opportunities automatically. In algorithmic trading, you typically make many trades based on quantitative factors to earn steady, small profits; this is unlike traditional manual trading where you make a few long-term investments based on market fundamentals and wait to earn a large return over time. Blockchains are often more suitable for algorithmic trading than manual trading, because the high volatility of cryptocurrencies in general makes them less suitable for traditional "buy and hold" investing; the XRP Ledger is particularly suited for algorithmic trading, for several reasons:
+
+- Decentralized exchange data is public and freely available. You don't need a subscription or special access to see what trades are available.
+- Trades settle in seconds, which is fast enough that you can do a lot of trading, but not so fast that you need a specialized network connection or extremely powerful computer. You can trade using a laptop and decent home internet connection.
+- Transaction fees on the XRP Ledger are very low, so you don't need much capital to start trading.
+
+## Trading Strategies
+
+Algorithmic trading can make profits through many different strategies; part of the challenge (or the fun) of algorithmic trading is designing and implementing your own unique approach. Other traders may be reluctant to share their strategies in detail, because that would lead to more people competing to take the same opportunities. From a high level, algorithmic trading approaches often fall into the following categories:
+
+- **Arbitrage:** Buying and immediately selling an asset to take advantages of price differences. This could involve finding sets of 3 or more assets where the prices aren't aligned, or using the XRP Ledger to move assets between private exchanges where the prices are different.
+- **Quantitative Trading:** Predicting and taking advantage of future price movements based on past price movements, outside data, or both. Examples include [candlestick patterns](https://blog.quantinsti.com/candlestick-patterns-meaning/), correlating an asset's price movements with other assets, and using sentiment analysis of social media.
+- **Front-running:** Taking advantage of pending trades, especially large ones, by buying and immediately reselling the assets those trades are buying. While the XRP Ledger's canonical ordering of transactions makes front-running difficult, it is not impossible.
```

Comment:
```suggestion
- **Front-running:** Taking advantage of pending trades, especially large ones, by buying and immediately reselling the assets those trades are buying. While the XRP Ledger's canonical ordering of transactions makes front-running difficult, it is not impossible.
```

Maybe elaborate on how this method opens opportunities to skim profits. Could be worth linking this bullet point and arbirtrage out to more detailed explanations.

Reply:
Additional note: isn't front-running frowned upon since the profits come at the expense of others? If so, maybe don't frame it as a way to make money?